k8s/namespaces/wa/kustomization.yaml

```diff
@@ -7,4 +7,5 @@ bases:
 - wa-task-management-api/wa-task-management-api.yaml
 - wa-task-configuration-api/wa-task-configuration-api.yaml
 - wa-proto-frontend/wa-proto-frontend.yaml
```

Comment:
is this supposed to be in prod?

Reply:
just preview

Reply:
can you remove then please, and move to the appropriate place

Reply:
@timja , we have confirmed with Ravi, we this service needs on Prod too. Thanks